platform_it/pom.xml

```diff
@@ -1002,7 +1002,7 @@
                                 <argument>-Dsplice.spark.kryoserializer.buffer=4m</argument>
                                 <argument>-Dsplice.spark.locality.wait=60s</argument>
                                 <argument>-Dsplice.spark.logConf=true</argument>
-                                <argument>-Dsplice.spark.master=yarn-client</argument>
+                                <argument>-Dsplice.spark.master=${splice.spark.master}</argument>
```

Comment:
yarn-client is deprecated in Spark 2.0+, no need to keep it.
spark.submit.deployMode may need to be explicitly specified.

Reply:
splice.spark.master is set to 'yarn', added splice.spark.submit.deployMode=client

Reply:
Do the same for the spliceFastSSL profile

Reply:
Sorry, did not notice second occurrence, fixed.